backend/db/refresh.py

```diff
@@ -83,8 +92,14 @@ def refresh_db(
             counts_docs()
             raise err
         finally:
+            # Update status vars
             update_db_status_var('last_refresh_exited', current_datetime(), local)
             update_db_status_var('refresh_status', 'inactive', local)
+            # Routine: Check for and resolve any open fetch failures
+            if resolve_fetch_failures_excess_items:
+                call_github_action('resolve-fetch-failures-excess-items')
+            if resolve_fetch_failures_0_members:
```

Comment:
`resolve_fetch_failures_0_members` is on.

But `resolve_fetch_failures_excess_items` is off due to outstanding issues #499 and #518. but it hardly matters, as it almost never happens.